src/components/Footer.ts

```diff
@@ -22,19 +25,55 @@ export class Footer extends LitElement {
         color: var(--bkd-func-fg-black);
       }
 
+      footer {
+        display: flex;
+        justify-content: space-between;
+      }
+
       .copyright {
         font-size: 0.8125rem;
         font-weight: 300;
         letter-spacing: 0.02rem;
         word-spacing: 0.05rem;
       }
 
+      .footer-nav a {
+        font-size: 14px;
+        font-weight: 400;
+        letter-spacing: 0.01rem;
+        word-spacing: 0.025rem;
+        line-height: 1.5;
+        color: var(--bkd-func-fg-black);
+        text-decoration: none !important;
+        display: inline-block;
+        margin: 0 calc(0.5vw + 30px) 0 0;
```

Comment:
Diese Grössen welche sich auf den ganzen Viewport beziehen, während der eigentlich Container in der Breite limitiert ist, sehe ich nicht ganz ein. Ich habe das beim Header nicht übernommen. Würde hier einfach einen passenden, fixen Pixelwert nehmen.

Was noch fehlt, ist dieses `margin-right` beim letzten Element wieder wegnehmen, ansonsten ist der Abstand am Rand des Footers nicht gleich gross auf beiden Seiten (in ihren Beispielen haben sie das gemacht):

![image](https://github.com/bkd-mba-fbi/evento-portal/assets/66047/8a1f9a7b-afbd-4a98-8bc2-5af7c197a4d5)

Vielleicht könnte man auf der `.footer-nav` auch gleich ein Flexbox machen mit einer entsprechenden `gap`. Oder sonst `.footer-nav a + .footer-nav a { margin-left: ...}`.